src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -295,18 +295,17 @@ const Analyses = _.flow(
     Utils.withBusyState(setBusy)
   )(async files => {
     try {
-      await Promise.all(_.map(async file => {
```

Comment:
Question about the async removal here, was there a particular reason for it being there in the first place?

Reply:
I believe it was there due to line 306
`const contents = await Utils.readFileAsText(file)`